extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/HueBindingConstants.java

```diff
@@ -51,4 +56,7 @@
     // Light config properties
     public static final String LIGHT_ID = "lightId";
 
+    // Added for the N-UPNP discovery
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UID = Collections.singleton(THING_TYPE_BRIDGE);
```

Comment:
This constant should rather be defined within `HueBridgeNupnpDiscovery`. Note that we e.g. also have `HueLightHandler.SUPPORTED_THING_TYPES`.